apps/worker/src/app/workflow/usecases/process-unsnooze-job/process-unsnooze-job.usecase.ts

```diff
@@ -56,6 +56,8 @@ export class ProcessUnsnoozeJob {
           $set: {
             snoozedUntil: null,
             createdAt: new Date(),
+            read: false,
+            lastReadDate: null,
```

Comment:
Whenever notification gets automatically unsnoozed it should be mark as unread always